src/main/java/com/epam/page/object/generator/adapter/classbuildable/SiteClassBuildable.java

```diff
@@ -0,0 +1,63 @@
+package com.epam.page.object.generator.adapter.classbuildable;
+
+import static com.epam.page.object.generator.util.StringUtils.firstLetterUp;
+import static com.epam.page.object.generator.util.StringUtils.splitCamelCase;
+import static javax.lang.model.element.Modifier.PUBLIC;
+import static javax.lang.model.element.Modifier.STATIC;
+
+import com.epam.jdi.uitests.web.selenium.elements.pageobjects.annotations.JPage;
+import com.epam.jdi.uitests.web.selenium.elements.pageobjects.annotations.JSite;
+import com.epam.page.object.generator.adapter.annotation.AnnotationMember;
+import com.epam.page.object.generator.adapter.annotation.IJavaAnnotation;
+import com.epam.page.object.generator.adapter.filed.IJavaField;
+import com.epam.page.object.generator.adapter.annotation.JavaAnnotation;
+import com.epam.page.object.generator.adapter.filed.JavaField;
+import com.epam.page.object.generator.adapter.javaclass.IJavaClass;
+import com.epam.page.object.generator.builder.JavaClassBuilder;
+import com.epam.page.object.generator.model.WebPage;
+import java.util.ArrayList;
+import java.util.List;
+import javax.lang.model.element.Modifier;
+
+public class SiteClassBuildable implements JavaClassBuildable {
+
+    private List<WebPage> webPages;
+
+    public SiteClassBuildable(List<WebPage> webPages) {
+        this.webPages = webPages;
+    }
+
+    @Override
+    public IJavaAnnotation buildAnnotation() {
+        List<AnnotationMember> annotationMembers = new ArrayList<>();
+        annotationMembers.add(new AnnotationMember("value", "$S", webPages.get(0).getDomainName()));
```

Comment:
Do we always have the 0 element in webPages list? do not we need the check?

Reply:
At the moment, we didn't implement situations when we have many domains, so we can take only first domain name.